pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -37,6 +39,10 @@ python3.pkgs.buildPythonApplication rec {
     hidapi=${ hidapi }/lib/
     test -d $hidapi || { echo "ERROR: $hidapi doesn't exist, please update/fix this build expression."; exit 1; }
     sed -i -e "s|libhidapi|$hidapi/libhidapi|" src/hhd/controller/lib/hid.py
+
+    # The generated udev rules point to /bin/chroot, which does not exist in NixOS
+    chmod=${ coreutils }/bin/chmod
```

Comment:
Should be updated to use toybox.

Reply:
Thanks to pointing that out, didn't realized it :sweat_smile: 

Btw are we using toolbox cause it's preferred by nixpkgs or cause it's already there? (or both idk)

Reply:
IDK if it matters.  Just suggesting to use it here because it was already in there.

(I think I chose it originally because it's a small dependency and I'd used it before to access POSIX commands.)